thunder/torch/__init__.py

```diff
@@ -2687,6 +2687,23 @@ def clone(a: TensorProxy, *, memory_format=torch.preserve_format) -> TensorProxy
 register_method("clone", clone)
 
 
+@torchsymbol(torch.nn.functional.glu, is_method=False)
+def glu(a: TensorProxy, /, dim: int = -1):
+    utils.check(
```

Comment:
I don't think this needs to check for the dim being within the expected range, but if you'd like to then maybe just call `canonicalize_dim`?

https://github.com/Lightning-AI/lightning-thunder/blob/15c48efc7fa2befd628ffa0e86a3e2d17816b9de/thunder/core/utils.py#L585